device/arista/x86_64-arista_7280r4_32qf_32df/Arista-7280R4-32QF-32DF-64O/gearbox_config.json

```diff
@@ -11,6 +11,7 @@
       "phy_access": "mdio",
       "bus_id": 0,
       "context_id": 1,
+      "macsec_supported": false,
```

Comment:
A related comment @arista-nwolfe, we need the flag macsec_enabled in the platform_env.conf in the parent platform dir : arista_7280r4_32qf_32df